src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+  const getSum = function(firstOperand) {
+    const secondNumber = function(secondOperand) {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
+        return `Bzzz... Error!`;
+      }
+
+      return firstOperand + secondOperand;
+    };
+
+    return secondNumber;
+  };
```

Comment:
you can return both of these functions right away, without creating variables for them